birdhouse/config/twitcher/default.env

```diff
@@ -2,6 +2,7 @@
 
 export TWITCHER_LOG_LEVEL=INFO
 export VERIFY_SSL="true"
+export TWITCHER_VERSION=0.8.0
```

Comment:
Sadly no, because Twitcher uses the Magpie-build version that matches with it.
This was only for the CanarieAPI config, but that is not very robust either, as we must remember to update it along Magpie (which we won't 😅).

I'll see if I can parse the Magpie Dockerfile link directly to retrieve it instead.